gcp/modules/gcp-stackdriver-monitoring/main.tf

```diff
@@ -2,15 +2,20 @@ terraform {
   backend "gcs" {}
 }
 
+variable "env" {}
 variable "nonce" {}
 variable "domain_name" {}
 variable "project_id" {}
 variable "serviceaccount_key" {}
+variable "auth_user_email" {}
 
-# Terragrunt variable
+# Terragrunt variables
+
+variable "notification_email" {}
 variable "ssl_enabled_uptime_checks" {}
 
 # Enables debug mode when TF_VAR_stackdriver_debug is not empty
+
```

Comment:
Extra newline?

Reply:
Added to [match the style](https://github.com/gpii-ops/gpii-infra/pull/241#discussion_r242371032).